textpipe/operation.py

```diff
@@ -346,10 +346,36 @@ class GensimTextRank(Operation):
     >>> doc = Doc(text)
     >>> GensimTextRank(ratio=0.2)(doc)
     ["She's crying with all her might and main, And she won't eat her dinner - rice pudding again - What is the matter with Mary Jane?", "She's perfectly well and she hasn't a pain, And it's lovely rice pudding for dinner again!"]
+    >>> GensimTextRank(word_count=20)(doc)
+    ["She's crying with all her might and main, And she won't eat her dinner - rice pudding again - What is the matter with Mary Jane?"]
+    >>> GensimTextRank(word_count=10)(doc)
+    []
     """
 
     def __init__(self, **kwargs):
         self.kwargs = kwargs
 
     def __call__(self, doc, **kwargs):
         return doc.generate_textrank_summary(**self.kwargs)
+
+
+class LeadBaseline(Operation):
```

Comment:
Rename to `LeadSentences`? Whether it is a baseline depends on the application, right?